rt_gene/scripts/estimate_blink_standalone.py

```diff
@@ -23,74 +23,67 @@ class BlinkEstimatorImagePair(BlinkEstimatorStandalone):
     def __init__(self, model_path, threshold, input_size, viz):
         super(BlinkEstimatorImagePair, self).__init__(model_path, threshold, input_size, viz)
 
-    def estimate(self, images):
-        right_eyes = [images[0]]
-        left_eyes = [images[1]]
-        probs, blinks = self.predict(right_eyes, left_eyes)
+    def estimate(self, left, right):
+        right_image = [self.load_img(right)]
+        left_image = [self.load_img(left)]
+        probs, blinks = self.predict(left_image, right_image)
 
         if self.viz:
-            #TODO: viz on eye images?
-            viz_img = self.overlay_prediction_over_img(right_eyes[0], blinks[0])
-            cv2.imshow('image pair vis', viz_img)  
+            pair_img = np.concatenate((right_image[0], left_image[0]), axis=1)
+            viz_img = self.overlay_prediction_over_img(pair_img, blinks[0])
+            cv2.imshow('image pair visualisation', viz_img)  
         return probs, blinks
 
-class BlinkEstimatorPathPair(BlinkEstimatorStandalone):
+
+class BlinkEstimatorFolderPair(BlinkEstimatorStandalone):
     def __init__(self, model_path, threshold, input_size, viz):
-        super(BlinkEstimatorPathPair, self).__init__(model_path, threshold, input_size, viz)
-        
-    def estimate(self, eye_paths):
-        right_eyes = [self.load_img(eye_paths[0])]
-        left_eyes = [self.load_img(eye_paths[1])]
-        probs, blinks = self.predict(right_eyes, left_eyes)
+        super(BlinkEstimatorFolderPair, self).__init__(model_path, threshold, input_size, viz)
 
-        if self.viz:
-            viz_img = self.overlay_prediction_over_img(right_eyes[0], blinks[0])
-            cv2.imshow('image pair vis', viz_img) 
-        return probs, blinks
+    def estimate(self, left, right):
+        left_images = []
+        right_images = []
+        
+        for left_image_name in listdir(left):
+            left_image_path = left + '/' + left_image_name
+            left_images.append(self.load_img(left_image_path))
 
-class BlinkEstimatorFolder(BlinkEstimatorStandalone):
-
-    def estimate(self, folder_path):
-        right_eyes = []
-        left_eyes = []
-        for file_name in [f for f in listdir(folder_path) if 'right' not in f]:
-            right_eye_path = folder_path + '/' + file_name.replace('left', 'right')
-            left_eye_path = folder_path + '/' + file_name
-            right_eyes.append(self.load_img(right_eye_path))
-            left_eyes.append(self.load_img(left_eye_path))
-        probs, blinks = self.predict(right_eyes, left_eyes)
+        for right_image_name in listdir(right):
+            right_image_path = right + '/' + right_image_name
+            right_images.append(self.load_img(right_image_path))
```

Comment:
It seems like the `flip` is missing in the `standalone` version - correct me if wrong though.